dice/deck.go

```diff
@@ -36,3 +36,7 @@ func (d *Deck) DealDice(num_dice int) (Dices, error) {
 		return nil, errors.New("Not enough dice in deck!")
 	}
 }
+
+func (d *Deck) AppendDice(new_dice Dice) {
```

Comment:
I think naming it AddDice would fit better here...

deck.AddDice(...)
deck.DealDice(...)

It sounds more like actual operations that can be done on a physical deck of dice